crates/client/src/serialize/txt/zone.rs

```diff
@@ -139,15 +139,16 @@ impl Parser {
         &mut self,
         lexer: Lexer,
         origin: Option<Name>,
+        class: Option<DNSClass>,
     ) -> ParseResult<(Name, BTreeMap<RrKey, RecordSet>)> {
         let mut lexer = lexer;
         let mut records: BTreeMap<RrKey, RecordSet> = BTreeMap::new();
 
         let mut origin: Option<Name> = origin;
+        let mut class: Option<DNSClass> = class;
```

Comment:
In the future, please don't mix unrelated changes, in this case, changing the order of the declarations here vs setting it to the argument value, as it makes it harder to review your changes.

Reply:
I'll keep that in mind. Thanks for the feedback!